### DISCUSSION TITLE
Dark theme for map

### DISCUSSION THREAD

**Discussion by alx-xlx** [Freika/dawarich#545] [Q&A]
Is there any option to make the map graphics seemless with the rest of the dawarich UI.

It would be awesome to have one bu no pressure.

**Comment by Freika** [comment]
Currently there is no dark map layers in Dawarich, but I hope to fix this

**Comment by js94x** [comment]
I am also interested in a native dark mode and tried something around. I would like to achieve a similar map style like:
https://github.com/remisalmon/Strava-local-heatmap/blob/master/images/heatmap.png 
or 
https://www.strava.com/maps/global-heatmap

With the help of the blog article and a browser addon, I managed to create a "hacky" dark mode for Dawarich

https://dev.to/deepakdevanand/leaflet-map-dark-theme-5ej0
https://chromewebstore.google.com/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne?pli=1

Adding this custom css in Stylus for the Dawarich domain:
```
    .leaflet-layer, .leaflet-control-zoom-in, .leaflet-control-zoom-out, .leaflet-control-attribution {
        filter: invert(100%) hue-rotate(180deg) brightness(95%) contrast(90%);
    }
```
resulted in this map:

<img width="1178" alt="Screenshot 2025-01-31 at 11 06 11" src="https://github.com/user-attachments/assets/e4c7c805-92f4-4780-91ff-612a14194c73" />

Now I am wondering which CSS class must be modified to change the color of the polyline from blue to orange

**Reply by Freika** [reply]
The polyline color is being set in js, not css as of now, maybe it will be changed later. Nice trick!
